src/components/Navigation/Navigation.stories.ts

```diff
@@ -0,0 +1,59 @@
+// Button.stories.ts
+
+import type { Meta, StoryObj } from "@storybook/react";
+import {
+  reactRouterParameters,
+  withRouter,
+} from "storybook-addon-remix-react-router";
+import Navigation from "./Navigation.tsx";
+
+const meta: Meta<typeof Navigation> = {
+  title: "Example/Navigation",
```

Comment:
This should be Components/Navigation

Reply:
why should it be components/navigation, we have stored the story file inside the same folder as navigation component, is this not acceptable? Would you like us to create a new stories folder to store stories in?